packages/client/src/components/body/body.component.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+import './body.style.css';
+import WomenAccessories from '../../assets/women-accessories.png';
```

Comment:
Hi Haile I can see that you put the images in client/src/assets and then you import them here. 
an easier way to do it, is to use the public folder in the client and then in assets you'd create a folder for ur images 
then you can access it from any folder you are without importing them like that (src="/public/assets/foldername/glass.png")